tests/ZiraLink.Tests/Tools/TestTools.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.EntityFrameworkCore;
+
+using Moq;
+
+using ZiraLink.Api.Application;
+using ZiraLink.Domain;
+using ZiraLink.Domain.Enums;
+
+namespace ZiraLink.Tests.Tools
+{
+    public class TestTools
+    {
+
+        public static DbContextOptions<AppDbContext>? contextOptions;
+        public static AppDbContext? _dbContext;
+        public static long customerId = 1;
```

Comment:
Do we need this field?

Reply:
Yes, we need this parameter to use everywhere. If we don't need to this parameter, I will remove it.

Reply:
I removed it.